docker/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM centos:7
```

Comment:
Why centos specific stuff in `docker/Dockerfile`?

Reply:
It's just the base image we are going to use at least for the time being.